platforms/bukkit/src/main/java/com/pg85/otg/bukkit/util/NBTHelper.java

```diff
@@ -1,300 +1,331 @@
 package com.pg85.otg.bukkit.util;
 
+import java.lang.reflect.Field;
```

Comment:
If there's any files where only the formatting was changed, can you revert those?